src/types/index.ts

```diff
@@ -6,9 +6,14 @@ export * from './register'
 export * from './resolved'
 export * from './resolvedQuery'
 export * from './routeMatchRule'
+export * from './routeWithParams.spec-d'
 export * from './routeWithParams'
 export * from './router'
+export * from './routerPush'
+export * from './routerReplace'
+export * from './routerRoute.spec-d'
```

Comment:
```suggestion
export * from './routeWithParams'
export * from './router'
export * from './routerPush'
export * from './routerReplace'
```

Reply:
thank you. been meaning to update my barrel generator extension to exclude `.spec-d`, just did that now